### ISSUE TITLE
ask: support notepad++ key bindings extension

### ISSUE THREAD

**Issue by chrisdias** [microsoft/azuredatastudio#8728]
copying over from the vscode repo

from @denvro, issue # https://github.com/microsoft/vscode-notepadplusplus-keybindings/issues/13

_This is a product request in stead of an issue. Could you please also make this extension compatible/available with Azure Data Studio? As most of the VSIX extensions are already_

**Comment by alanrenmsft** [comment]
@chrisdias Does the extension work when you download the vsix and install it in Azure Data Studio? or you are asking us to make it available in the marketplace of Azure Data Studio. please clarify.

**Comment by denvro** [comment]
No, downloading de vsix and installing it manually in Azure Data Studio doesn't work. You get an error message that it's not compatible. Most of the VSCode vsix extension you can however install manually by downloading them. It would of course be a nice to have that, when it's compatible, it's availble in the ADS marketplace.

**Comment by alanrenmsft** [comment]
this is caused by the version requirement in the extension's package.json, ADS's vscode version is 1.38.0, but the extension is requiring 1.41.0 or later.
@denvro, to unblock you for now, you could update the vscode engine requirement in package.json. see below, and you will be able to install it in ADS.

````JSON
    "engines": {
        "vscode": "^1.37.0"
    },
````

@anthonydresser have we stopped updating the "vscodeVersion" attribute in the product.json when doing vscode merges?

**Comment by Charles-Gagnon** [comment]
I don't think that was ever part of the VS Code merge process. I started doing it as part of the release process but I guess that hasn't been kept up with the past few releases. I'll go bump that up now since I'm doing the release this time around.

**Comment by alanrenmsft** [comment]
@denvro if you are not the author of the notepad++ keybinding extension, you can download the vsix file and change the file extension from vsix to zip and unzip it, inside it you will be able to find the package.json and make the change i mentioned above. after that you can zip the contents (don't zip the folder directly) and change the file extension from zip to vsix. you should be able to install it now.
